docs/sources/upgrading/_index.md

```diff
@@ -35,7 +35,26 @@ The output is incredibly verbose as it shows the entire internal config struct u
 
 ### Loki
 
+#### Index gateway shuffle sharding
+
+The index gateway now supports shuffle sharding of index data when running in
+"ring" mode. The index data is sharded by tenant where each tenant gets
+assigned a sub-set of all available instances of the gateway ring.
+
+In case you configured a high replication factor to accommodate for load, since
+in the past this was the only option to give a tenant more instances for
+querying, you should consider reducing the replication factor to meaningful
+value for replication (e.g. from 12 to 3) and instead set the shard factor for
+individual tenants as required.
+
+If the global shard factor (no per-tenant) is 0 (default value), the global
+shard factor of is set to replication factor. It can still be overwritten per
```

Comment:
"shard factor of is set to replication "  Same comment as above, either there is a word missing here or delete the word "of".